spec/dealership_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rspec'
+require './lib/car'
+require './lib/dealership'
+
+RSpec.describe Dealership do
+  describe 'Iteration II' do
+  let(:car_1) {Car.new("Ford Mustang", 1500, 36)}
+  let(:car_2) {Car.new("Toyota Prius", 1000, 48)}
+  let (:dealership) {Dealership.new("Acme Auto", "123 Main Street")}
```

Comment:
I like the use of `let` up here so that you could reuse the car_1, car_2, and dealership objects across multiple tests.